contentreview-impl/impl/src/java/org/sakaiproject/contentreview/impl/advisors/SiteToolAdvisor.java

```diff
@@ -26,6 +26,13 @@ public boolean siteCanUseReviewService(Site site) {
         return false;
     }
 
+	public boolean siteCanUseLTIReviewService(Site site) {
+        for(String toolId : contentReviewTools)
```

Comment:
I just copied the method that appeared in there, as this class doesn't appear on TII 10.x branch. Not sure if it's an Oxford class or if it's used?